### DISCUSSION TITLE
Use Blade components to render errors pages

### DISCUSSION THREAD

**Discussion by freekmurze** [laravel/framework#38786] [Ideas]
In the [default error page views](https://github.com/laravel/framework/blob/8.x/src/Illuminate/Foundation/Exceptions/views/500.blade.php), `@extend` and. `@section` are being used. 

It would be nice if that would be replaced by Blade components.

**Comment by ousid** [comment]
Something like this?
```blade
<x-error-page title="Error Page" code="500" message="Error Page"/>
```
